velox/exec/Spiller.cpp

```diff
@@ -421,13 +429,23 @@ void Spiller::ensureSorted(SpillRun& run) {
   uint64_t sortTimeNs{0};
   {
     NanosecondTimer timer(&sortTimeNs);
-    gfx::timsort(
-        run.rows.begin(),
-        run.rows.end(),
-        [&](const char* left, const char* right) {
-          return container_->compareRows(
-                     left, right, state_.sortCompareFlags()) < 0;
-        });
+    const auto prefixSortRequiredBytes = PrefixSort::maxRequiredBytes(
+        memory::spillMemoryPool(),
+        container_,
+        state_.sortCompareFlags(),
+        state_.prefixSortConfig());
```

Comment:
Let's have a separate config for maxNormalizedKeySize for spill so that we can selective disable prefix sort for spill. Thanks!